alf/environments/suite_dmlab.py

```diff
@@ -198,21 +200,21 @@ def load(scene,
             https://github.com/deepmind/lab/tree/master/game_scripts/levels
         discount (float): Discount to use for the environment.
         frame_skip (int): the frequency at which the agent experiences the game
-        gym_env_wrappers (list): Iterable with references to wrapper classes to use
-            directly on the gym environment.
-        torch_env_wrappers (list): Iterable with references to wrapper classes to use
-            directly on the torch environment.
+        gym_env_wrappers (Iterable): Iterable with references to gym_wrappers, 
+            classes to use directly on the gym environment.
+        torch_env_wrappers (Iterable): Iterable with references to torch_wrappers 
+            classes to use on the torch environment.
         wrap_with_process (bool): Whether wrap env in a process
         max_episode_steps (int): max episode step limit
     Returns:
-        A PyEnvironmentBase instance.
+        A TorchEnvironment instance.
     """
     _unwrapped_env_checker_.check_and_update(wrap_with_process)
 
     if max_episode_steps is None:
         max_episode_steps = 0
 
-    def env_ctor():
+    def env_ctor(env_id=None):
```

Comment:
It will potentially be used in process environment https://github.com/HorizonRobotics/alf/blob/PR_suite_wrappers/alf/environments/process_environment.py#L102

Reply:
Why does the underlying environment have to know the id? It's something that our training code needs.